3p/remote.html

```diff
@@ -4,9 +4,22 @@
 <meta name="robots" content="noindex">
 <script>
 (function() {
-var v = location.search.substr(1);
 if (!(/^\d+(-canary)?$/.test(v))) return;
-var u = 'https://3p.ampproject.net/'+encodeURIComponent(v)+'/f.js';
+var c = window.AMP_CONFIG = window.AMP_CONFIG || {};
+var context;
+try {
+  context = JSON.parse(location.hash.substring(
+    location.hash.indexOf('{'),
+    location.hash.lastIndexOf('}') + 1
+  ))._context;
+}
+catch (e) { context = { AMP_CONFIG: {} }; }
+for (var key in context.AMP_CONFIG)
+  if (c[key] === undefined) c[key] = context.AMP_CONFIG[key]
+var v = context.mode.version;
+var u = context.mode.localDev || context.mode.test
+  ? './integration.js'
+  : c.thirdPartyUrl+'/'+encodeURIComponent(v)+'/f.js';
```

Comment:
hmm... `remote.html` is an example in case publisher wants to host the bootstrap html themselves. Normally they should just be testing against `frame.max.html`. 

@jrf0110 what's the use case for `remote.html` do choose env at runtime?